protoc-cli/src/test/resources/testng.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="ballerina-test-suite">
+    <listeners>
+        <listener class-name="io.ballerina.protoc.util.TestNGListener" />
+    </listeners>
+    <test name="protoc-cli-test-suite" preserve-order="true" parallel="false">
```

Comment:
since we dont have native tests, do we need these?

Reply:
Removed in [6b9c020](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/6b9c020e52ef52c7818a676fcccfbccc8630f1d8)